uint32 msg_pub_cnt
uint32 msg_sub_cnt

float64 Lane_Curvature
float64 Lane_Head

uint32 LeftLane_Confidence
uint32 LeftLane_LDW_Available
uint32 LeftLane_Type
float64 LeftLane_Distance

uint32 RightLane_Confidence
uint32 RightLane_LDW_Available
uint32 RightLane_Type
float64 RightLane_Distance

uint32 HW_Valid
float64 HW_Measurement
uint32 HW_Level

uint32 Error_ON
uint32 Error_code

uint32 LDW_ON
uint32 LDW_Left
uint32 LDW_Right

uint32 Change_LeftLane
uint32 Change_RightLane

uint32 FCW_ON
uint32 Peds_FCW
uint32 Peds_DZ

uint32 TSR_En

uint32[7] Sign_Type
uint32[7] S_Sign_Type
float64[7] Sign_X
float64[7] Sign_Y
float64[7] Sign_Z
uint32[7] Filter_Type

uint32[4] Sign
uint32[4] S_Sign

uint32 CA

float64 Yaw
float64 Pitch

uint32 Signal_Brake
uint32 Signal_Left
uint32 Signal_Right
uint32 Wiper
uint32 Beam_Low
uint32 Beam_High
uint32 Speed
